### PR TITLE
Year is fixed in About dialog

### PR DESCRIPTION
**Pull request by ARYA-BHARDWAJ**
This is regarding issue no #1205 
- change year 2019 to 2020

### CODE REVIEW

**Comment by ARYA-BHARDWAJ** [comment]
@ice0 please have a look ..

File: synfig-studio/src/gui/dialogs/about.cpp [hunk 1]
```diff
@@ -226,4 +226,4 @@ About::About()
 
 void About::close(int){
 	hide();
-}
+}
```
Comment (ice0) [inline synfig-studio/src/gui/dialogs/about.cpp]: Please add a newline in the end of file

**Comment by ARYA-BHARDWAJ** [comment]
@ice0 review now

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**